ibis/backends/dask/udf.py

```diff
@@ -116,10 +125,30 @@ def lazy_agg(*series: pandas.Series):
         # Depending on the type of operation, lazy_result is a Delayed that
         # could become a dd.Series or a dd.core.Scalar
         if isinstance(op, ops.AnalyticVectorizedUDF):
-            meta = pandas.Series(
-                [], name=args[0].name, dtype=op._output_type.to_dask()
-            )
+            if isinstance(op._output_type, dt.Struct):
+                meta = make_struct_op_meta(op)
+            else:
+                meta = make_meta_series(
+                    dtype=op._output_type.to_dask(), name=args[0].name,
+                )
             result = dd.from_delayed(lazy_result, meta=meta)
+
+            if args[0].known_divisions:
+                if not len({a.divisions for a in args}) == 1:
+                    raise ValueError(
+                        "Mixed divisions passed to AnalyticVectorized UDF"
```

Comment:
hit in tests?

Reply:
No - I'm honestly not really sure how you'd hit this or cause this to occur. Generally any args here are coming from the same dataframe, so you'd need to somehow repartition before hitting this function. 

I can write a very contrived test that would just import this directly?